src/utils.ts

```diff
@@ -27,6 +28,10 @@ export class Utils {
     private static readonly LATEST_RELEASE_VERSION: string = '[RELEASE]';
     // The default server id name for separate env config
     public static readonly SETUP_JFROG_CLI_SERVER_ID: string = 'setup-jfrog-cli-server';
+    // Directory name which holds markdown files for the Workflow summary
+    private static readonly JOB_SUMMARY_DIR_NAME: string = 'jfrog-command-summary';
+    // Workflow summary section files
+    public static JOB_SUMMARY_MARKDOWN_SECTIONS_NAMES: string[] = ['upload', 'build-info', 'security'];
```

Comment:
I think this change requires moving the Section enum up and renaming it to MarkdownSection.
```suggestion
    public static JOB_SUMMARY_MARKDOWN_SECTIONS_NAMES: MarkdownSection[] = [MarkdownSection.Upload, MarkdownSection.BuildInfo, MarkdownSection.Security];
```